indico/modules/legal/templates/manage_messages.html

```diff
@@ -7,12 +7,25 @@
 
 {% block content %}
     {% call simple_form(form, back_button=false) %}
-        {{ form_rows(form, skip=('tos_url', 'tos', 'privacy_policy_url', 'privacy_policy')) }}
+        {{ form_rows(
+               form,
```

Comment:
not a fan of this wrapping style tbh :/

Reply:
Say thank you to `black` which formatted this block (and also is responsible for the double quotes :). I'll try a different approach.